src/components/screens/error-page.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement } from "react";
+import { useRouteError } from "react-router-dom";
+
+/**
+ * Error page for displaying error status text and message
+ * 
+ * @returns ReactElement
+ */
+export default function ErrorPage (): ReactElement {
+  const error: any = useRouteError();
```

Comment:
```suggestion
  const error: unknown || something else?  = useRouteError();
```
We struggled for a while with finding the correct type for this instance, so I'd say this is low priority.